UI/Pages/TuitionPartner.cshtml

```diff
@@ -4,260 +4,289 @@
+                    {
+                        <li>@item</li>
+                    }
+                </ul>
+            </govuk-summary-list-row-value>
+        </govuk-summary-list-row>
+        <govuk-summary-list-row>
+            <govuk-summary-list-row-key>
+                Type of tuition
+            </govuk-summary-list-row-key>
+            <govuk-summary-list-row-value>
+                <ul class="govuk-list" data-testid="type-of-tuition">
+                    @foreach (var item in Model.Data.TuitionTypes)
+                    {
+                        <li>@item</li>
+                    }
+                </ul>
+            </govuk-summary-list-row-value>
+        </govuk-summary-list-row>
+        <govuk-summary-list-row>
+            <govuk-summary-list-row-key>
+                Organisation type
+            </govuk-summary-list-row-key>
+            <govuk-summary-list-row-value data-testid="organisation-type">
+                @Model.Data.LegalStatus
+            </govuk-summary-list-row-value>
+        </govuk-summary-list-row>
+    </govuk-summary-list>
+</div>
+
+<table class="govuk-table" data-testid="locations-covered-table" show-if="@Model.Data.LocalAuthorityDistricts.Any()">
+    <caption class="govuk-table__caption govuk-table__caption--m">Locations covered</caption>
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Region</th>
+        <th scope="col" class="govuk-table__header">LAD code</th>
+        <th scope="col" class="govuk-table__header">LAD name</th>
+        <th scope="col" class="govuk-table__header">In school</th>
+        <th scope="col" class="govuk-table__header">Online</th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+    @foreach (var item in Model.Data.LocalAuthorityDistricts)
+    {
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">@item.Region</td>
+            <td class="govuk-table__cell">@item.Code</td>
+            <td class="govuk-table__cell">@item.Name</td>
+            <td class="govuk-table__cell">@(item.InSchool ? "Yes" : "")</td>
+            <td class="govuk-table__cell">@(item.Online ? "Yes" : "")</td>
+        </tr>
+    }
+    </tbody>
+</table>
+
+<div>
+    <h2 class="govuk-heading-m">Tuition cost information</h2>
+
+    <p class="govuk-body">
+        Each pupil premium eligible student will receive £162. This funding cost has been calculated so you will have 60% of the cost covered for up to £18 per pupil per hour. For help understanding your funding, you can see our information on <a href="/funding-and-reporting" class="govuk-link" data-testid="funding-guidance-1">funding and reporting</a>.
+    </p>
+
+    <div>
+        <govuk-details data-testid="payment-details">
             <govuk-details-summary>
-                What is a quality assured tuition partner?
+                <govuk-details-text>
```

Comment:
I don't this is needed?  The existing markup already appears to render as https://design-system.service.gov.uk/components/details/

Reply:
Okay, on Rider I believe it flags a warning about it which was why I thought to change it.